addons/web/static/src/views/fields/fields.scss

```diff
@@ -93,6 +93,7 @@
     // Default display and alignment of widget and internal <input/>
     text-align: inherit;
     display: var(--fieldWidget-display, inline-block);
+    margin-bottom: var(--fieldWidget-margin-bottom, initial);
```

Comment:
`initial`: why this fallback value it's needed? 🤔

Reply:
It's absolutely useless sorry about that 😅 If no margin is set it would be initial value anyways, if a margin-bottom is set it would override the property, and if the custom property is set it would take the CPP value.